kernel/src/table_changes/mod.rs

```diff
@@ -255,8 +254,8 @@ fn check_cdf_table_properties(table_properties: &TableProperties) -> DeltaResult
 /// Ensures that Change Data Feed is supported for a table with this [`Protocol`] .
 /// See the documentation of [`TableChanges`] for more details.
 fn ensure_cdf_read_supported(protocol: &Protocol) -> DeltaResult<()> {
-    static CDF_SUPPORTED_READER_FEATURES: LazyLock<HashSet<ReaderFeature>> =
-        LazyLock::new(|| HashSet::from([ReaderFeature::DeletionVectors]));
+    static CDF_SUPPORTED_READER_FEATURES: LazyLock<Vec<ReaderFeature>> =
+        LazyLock::new(|| Vec::from([ReaderFeature::DeletionVectors]));
```

Comment:
```suggestion
        LazyLock::new(|| vec![ReaderFeature::DeletionVectors]);
```
(several more below)